src/plugins/graph/models/graph-layer-model.ts

```diff
@@ -104,7 +104,6 @@ export const GraphLayerModel = types
         : undefined;
     },
     updateAdornments(resetPoints=false) {
-      console.log('updateAdornments for ', self.config.dataset?.id);
```

Comment:
This was originally my `console` statement - that got pushed to `main`, taking it out.